pkg/nmi/server/endpointpolicy.go

```diff
@@ -14,15 +14,34 @@ import (
 	"k8s.io/klog"
 )
 
+const (
+	InvalidOperation = "InvalidOperation"
+	NotFound         = "NotFound"
+)
+
+type endpointPolicyError struct {
+	errType string
+	err     error
+}
+
+func (e *endpointPolicyError) Error() string {
+	return fmt.Sprintf("%s: %v", e.errType, e.err)
```

Comment:
>v [](start = 26, length = 1)

%w here

Reply:
For fmt.Sprintf(), %w will hit the following error:
Sprintf call has error-wrapping directive %w

%w is good for for fmt.Errorf().